rune/libenclave/check.go

```diff
@@ -304,6 +310,20 @@ func ValidateEnclave(config *enclaveConfigs.EnclaveConfig) error {
 		return err
 	}
 
+	IsValidLogLevel := false
```

Comment:
Add a comment line here to describe these log level is inherited from runc --loglevel option.